Alfie/Packages/Core/Sources/Services/Configuration/ConfigurationService.swift

```diff
@@ -132,29 +132,25 @@ extension ConfigurationService {
     }
 
     private func appUpdateInfo(type: AppUpdateType) -> AppUpdateInfo? {
-        guard
-            let rawValue = providerValue(for: .appUpdate),
-            let appUpdate = ConfigurationValue(rawValue: rawValue)?.appUpdate,
-            let configuration: ConfigurationAppUpdateInfo = {
-                // swiftlint:disable vertical_whitespace_between_cases
-                switch type {
-                case .immediate:
-                    return appUpdate.requirements.immediate
-                case .flexible:
-                    return appUpdate.requirements.flexible
-                }
-                // swiftlint:enable vertical_whitespace_between_cases
-            }()
-        else {
-            return nil
-        }
+        guard let rawValue = providerValue(for: .appUpdate),
```

Comment:
Please keep each condition on new line
```suggestion
        guard 
            let rawValue = providerValue(for: .appUpdate),
```